ci/concourse/azure-template-updater.yml

```diff
@@ -12,6 +12,12 @@ resources:
       uri: ((gitrepo.url))
       branch: ((gitrepo.base_branch))
       private_key: ((gitrepo.private_key))
+  - name: git-repo-fortigate-autoscale-azure-updated
+    type: git
+    source:
+      uri: ((gitrepo.url))
+      branch: ((gitrepo.base_branch))
+      private_key: ((gitrepo.private_key))
```

Comment:
need private_key_passphrase (requires concourse 7.3.0+ or a custom git resource 1.13.0+)

Reply:
can create a new update for this in a separate change.